Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -1826,8 +1825,11 @@ LayoutUnit RenderFlexibleBox::computeCrossSizeForChildUsingContainerCrossSize(co
         // Let's compute the definite size value for the flex item (value that we can resolve without running layout).
         auto isHorizontal = isHorizontalFlow();
         auto size = isHorizontal ? style().height() : style().width();
-        ASSERT(size.isFixed());
```

Comment:
Why can this ASSERT be removed and not added to?

Reply:
Actually I forgot that these are not in release builds. Initially I thought it was a little much too for the synchronization between this method and childCrossSizeShouldUseContainerCrossSize as we implement more of the definite sizes, but now I realize it's probably fine. Will add to it.